SampleApp/SampleApp.iOS/SampleContentExtension/SampleContentExtension.csproj

```diff
@@ -107,4 +107,5 @@
     </Compile>
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\iOS\Xamarin.iOS.AppExtension.CSharp.targets" />
+  <Import Project="..\..\packages\urbanairship.ios.notificationcontentextension.14.2.3\build\urbanairship.ios.notificationcontentextension.targets" Condition="Exists('..\..\packages\urbanairship.ios.notificationcontentextension.14.2.3\build\urbanairship.ios.notificationcontentextension.targets')" />

```

Comment:
The user doesn't need to do anything special to make this happen, it will automatically get imported along with the package